### PR TITLE
TopoJSON Map Fixes

### PR DESCRIPTION
**Pull request by reb-dev**
This PR addresses multiple issues with TopoJSON maps, makes some revisions to the TopoJSONMap component doc page, and adds a few new examples to the demo app.

Changes:
- Fixed: when multiple TopoJSONMap components are on the same page, they share the same projection instance and cause inconsistent behavior when panning/zooming on any one of them (This is related to the change when we converted configs from classes to objects, sorry @rokotyan I found another one)
- Fixed: Zooming/panning when `mapFitToPoints` is enabled will reset the view entirely
- Fixed the issue where TopoJSONLayer property in LeafletMap component not taking effect
- If points are added/removed to/from the data and `mapFitToPoints` is enabled, the map will now refit to the new points (previously did nothing)
- Added the following countries that were missing from WorldMapTopoJSON:

| Name                                           |  Country Code |
|------------------------------- | --------------|
| Aruba                                          | AW           |
| Anguilla                                       | AI           |
| American Samoa                                 | AS           |
| Antigua and Barbuda                            | AG           |
| Bermuda                                        | BM           |
| Barbados                                       | BB           |
| Cook Islands                                   | CK           |
| Curaçao                                        | CW           |
| Cayman Islands                                 | KY           |
| Micronesia, Federated States of                | FM           |
| Guernsey                                       | GG           |
| Grenada                                        | GD           |
| Guam                                           | GU           |
| Heard Island and McDonald Islands              | HM           |
| Isle of Man                                    | IM           |
| Cocos Islands                                  | CC           |
| Christmas Island                               | CX           |
| British Indian Ocean Territory                 | IO           |
| Jersey                                         | JE           |
| Kiribati                                       | KI           |
| Saint Kitts and Nevis                          | KN           |
| Macao                                          | MO           |
| Monaco                                         | MC           |
| Maldives                                       | MV           |
| Marshall Islands                               | MH           |
| Malta                                          | MT           |
| Northern Mariana Islands                       | MP           |
| Montserrat                                     | MS           |
| Norfolk Island                                 | NF           |
| Niue                                           | NU           |
| Pitcairn                                       | PN           |
| Palau                                          | PW           |
| Saint Helena, Ascension and Tristan da Cunha   | SH           |
| San Marino                                     | SM           |
| Saint Pierre and Miquelon                      | PM           |
| Seychelles                                     | SC           |
| Turks and Caicos Islands                       | TC           |
| Tonga                                          | TO           |
| Saint Vincent and the Grenadines               | VC           |
| Virgin Islands, British                        | VG           |
| Virgin Islands, U.S.                           | VI           |
| Wallis and Futuna                              | WF           |

### CODE REVIEW

File: packages/dev/src/examples/maps/topojson/world-map/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <VisTopoJSONMap
+          topojson={WorldMapTopoJSON}
+          duration={0}
+          areaColor='red'
+          zoomExtent={[0.5, 100]}
+          events={{
+            [TopoJSONMap.selectors.feature]: {
+              // click: c => console.log(c),
+            },
+          }}
```
Comment (lee00678) [inline packages/dev/src/examples/maps/topojson/world-map/index.tsx:30]: Unused code?

**Review by lee00678 — CHANGES_REQUESTED** [review]
Thanks for the fixes! This is great. 

I just have one issue I've noticed. (pan or zoom on this map, the map disappears)

https://github.com/user-attachments/assets/81751b74-e5fd-482d-89b8-0457779f2f07

**Comment by rokotyan** [comment]
Thanks for the fixes @reb-dev!

**Comment by reb-dev** [comment]
> I just have one issue I've noticed. (pan or zoom on this map, the map disappears)

Can you try `npm run install:clean` and try the local build again? I can't seem to reproduce it on my end

**Comment by lee00678** [comment]
> Can you try `npm run install:clean` and try the local build again? I can't seem to reproduce it on my end

My bad, it's working after the clean install.

**Review by lee00678 — APPROVED** [review]

**Pull request merged**